main.tf

```diff
@@ -42,16 +42,14 @@ locals {
     "ecr:GetDownloadUrlForLayer",
   ]
 
-  ecr_push_only_actions = [
+  ecr_push_actions = [
     "ecr:CompleteLayerUpload",
     "ecr:InitiateLayerUpload",
     "ecr:ListImages",
     "ecr:PutImage",
     "ecr:UploadLayerPart",
   ]
 
-  ecr_push_actions = concat(local.ecr_push_only_actions, local.ecr_pull_actions)
-
```

Comment:
this is a breaking change and the commit message should state this.
in addition this is a change not "clean up code"